{"name": "邀请用户", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x0024A813+48355]\n\t(No symbol) [0x001DC4B1]\n\t(No symbol) [0x000E5358]\n\t(No symbol) [0x001109A5]\n\t(No symbol) [0x00110B3B]\n\t(No symbol) [0x0013E232]\n\t(No symbol) [0x0012A784]\n\t(No symbol) [0x0013C922]\n\t(No symbol) [0x0012A536]\n\t(No symbol) [0x001082DC]\n\t(No symbol) [0x001093DD]\n\tGetHandleVerifier [0x004AAABD+2539405]\n\tGetHandleVerifier [0x004EA78F+2800735]\n\tGetHandleVerifier [0x004E456C+2775612]\n\tGetHandleVerifier [0x002D51E0+616112]\n\t(No symbol) [0x001E5F8C]\n\t(No symbol) [0x001E2328]\n\t(No symbol) [0x001E240B]\n\t(No symbol) [0x001D4FF7]\n\tBaseThreadInitThunk [0x75C67D59+25]\n\tRtlInitializeExceptionChain [0x7736B74B+107]\n\tRtlClearBits [0x7736B6CF+191]\n\t(No symbol) [0x00000000]", "trace": "self = <new_admin_demo.test.test_user_manage.TestClass_usermanage object at 0x0000024D54C26E80>, driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae03fceff9df300a62be96e5eb35369c\")>\nget_data = {'case': {'keyword': '1921507475@qq.com,1234567'}}\n\n    @allure.story('账号管理')\n    @allure.title('邀请用户')\n    @pytest.mark.parametrize(\"get_data\", yaml.safe_load(open('../datafile/test_datatext/test__invite_user.yaml', encoding='utf-8')))\n    def test_invite_user(self,driver,get_data):\n        keyword=get_data[\"case\"]\n        input=keyword['keyword']\n        Invite_user=Locator_invite_user(driver)\n        # 检测用户页面漏翻译\n>       Invite_user.click_user()\n\ntest_user_manage.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n..\\pages\\page_newadmin.py:55: in click_user\n    self.click_i(self.user)\n..\\pages\\base_action.py:37: in click_i\n    ele = self.find_element(location)\n..\\pages\\base_action.py:23: in find_element\n    return wait.until(lambda driver: self.driver.find_element(local_by, local_value))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ae03fceff9df300a62be96e5eb35369c\")>, method = <function BaseAction.find_element.<locals>.<lambda> at 0x0000024D546D9670>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tGetHandleVerifier [0x0024A813+48355]\nE       \t(No symbol) [0x001DC4B1]\nE       \t(No symbol) [0x000E5358]\nE       \t(No symbol) [0x001109A5]\nE       \t(No symbol) [0x00110B3B]\nE       \t(No symbol) [0x0013E232]\nE       \t(No symbol) [0x0012A784]\nE       \t(No symbol) [0x0013C922]\nE       \t(No symbol) [0x0012A536]\nE       \t(No symbol) [0x001082DC]\nE       \t(No symbol) [0x001093DD]\nE       \tGetHandleVerifier [0x004AAABD+2539405]\nE       \tGetHandleVerifier [0x004EA78F+2800735]\nE       \tGetHandleVerifier [0x004E456C+2775612]\nE       \tGetHandleVerifier [0x002D51E0+616112]\nE       \t(No symbol) [0x001E5F8C]\nE       \t(No symbol) [0x001E2328]\nE       \t(No symbol) [0x001E240B]\nE       \t(No symbol) [0x001D4FF7]\nE       \tBaseThreadInitThunk [0x75C67D59+25]\nE       \tRtlInitializeExceptionChain [0x7736B74B+107]\nE       \tRtlClearBits [0x7736B6CF+191]\nE       \t(No symbol) [0x00000000]\n\nc:\\users\\user\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:90: TimeoutException"}, "parameters": [{"name": "get_data", "value": "{'case': {'keyword': '1921507475@qq.com,1234567'}}"}], "start": 1688032530089, "stop": 1688032561107, "uuid": "170d3079-f5eb-4468-ae09-4930162ce743", "historyId": "deafab78964a6be78194b51cfcab03a8", "testCaseId": "5a3acd59081a588b49d74a48bd4f20f2", "fullName": "test_user_manage.TestClass_usermanage#test_invite_user", "labels": [{"name": "story", "value": "账号管理"}, {"name": "suite", "value": "test_user_manage"}, {"name": "subSuite", "value": "TestClass_usermanage"}, {"name": "host", "value": "DESKTOP-8HRET3J"}, {"name": "thread", "value": "41200-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_user_manage"}]}